### ISSUE TITLE
CycleOSM and humanitarian unavailable.

### ISSUE THREAD

**Issue by WebMaster50** [osm-fr/infrastructure#285]
For a few days now, I have not been able to reach both the CycleOSM and Humanitarian layers on osm.org. After discussion at the Openstreetmap World discord server, it turns out that the servers IP banned me for no reason. We came to this conclusion because the layers work on a VPN. The layers just stay grey all the time while the OSM carto layer renders fine.

Help would be appriciated to resolve this issue. I almost never used these layers so I can't be because I overflooded the server.

**Comment by WebMaster50** [comment]
I just send an email to the adress you provided.

**Comment by Marc-marc-marc** [comment]
admin list didn't get it and I tested the admin list, it work

**Comment by WebMaster50** [comment]
Strange, it is send and I got a reply that it was forwarded to the moderators. Here is the truncated address: 192.168.0.xxx

**Comment by Marc-marc-marc** [comment]
oups sorry, I get it (was in the moderator queue)

**Comment by Marc-marc-marc** [comment]
the ip you sent is a private ip internal to your network.
to know the public ip (the one used to reach the server), you can for example use http://checkip.dyndns.org/

**Comment by WebMaster50** [comment]
My bad. 217.105.34.xxx

**Comment by cquest** [comment]
There is no ban on this IP range. We do not ban IPs unless they try to login on our server, only for security measures.

We have a rate limiting delaying display on tiles to limit mass downloads, but no full ban.

What do you mean by "the layer works on a VPN" ?

**Comment by WebMaster50** [comment]
Alright, just replied to your email but lets continue discussion here for now:

When I enable my vpn, I am able to view cyclOSM and humanitarian on OSM.org. Without a vpn, the maps just stays grey.

I already tried releasing and renewing the IP but that didn't work. Also, the IP gets resolved correctly. I also tested it in different browsers (Vivaldi, Brave and firefox) and the problem occured in all browsers. Nobody else I talked to were facing the same problems as me.
